invenio_matcher/models.py

```diff
@@ -27,7 +27,8 @@ class MatchResult(object):
 
     """Matcher - represent a result."""
 
-    def __init__(self, record, score):
+    def __init__(self, id_, record, score):
+        self.id = id_
```

Comment:
You mean not have id as part of the result object? Or where we get it from the record? 

I like the idea of having the id on the result object simply because it is something that will be often accessed by things. To get the id from ES is tied to ES implementation, so technically more stable. To get id from Record object is more tied to data model, so less stable/instance specific - unless we go deep inside record model object or pidstore ID eventually.